crypto/s2n_drbg.c

```diff
@@ -31,7 +31,7 @@
     acceptable in DRBG */
 int s2n_increment_drbg_counter(struct s2n_blob *counter)
 {
-    for (int i = counter->size - 1; i >= 0; i--) {
+    for (uint32_t i = counter->size - 1; i >= 0; i--) {
```

Comment:
Those sound more like `size_t` candidates?

Reply:
that's probably reasonable.  i can update all the for loops with a little diff sed magic...